src/main/java/hu/oe/nik/szfmv/automatedcar/AutomatedCar.java

```diff
@@ -8,46 +8,56 @@
 
 public class AutomatedCar extends WorldObject {
 
-    private PowertrainSystem powertrainSystem;
-    private SteeringSystem steeringSystem;
+    private final PowertrainSystem powertrainSystem;
+    private final SteeringSystem steeringSystem;
     private final VirtualFunctionBus virtualFunctionBus = new VirtualFunctionBus();
+    private final double speedMetersPerSeconds;
+
+    private final double wheelRadius;
 
     /**
      * Constructor of the AutomatedCar class
      *
      * @param x             the initial x coordinate of the car
      * @param y             the initial y coordinate of the car
-     * @param imageFileName name of the image file used displaying the car on the course display
+     * @param imageFileName name of the image file used displaying the car on the
```

Comment:
Tab van a fájlban. A behúzás 4 darab szóköz.

Reply:
@SzFMV2018-Osz/team-b4 Nem úgy látom mintha bárki is foglalkozott volna ezzel az issue-val.
Szeretném felhívni a figyelmet arra, hogy az értékelés feltétele a masterbe kerülés.